.github/workflows/build.yml

```diff
@@ -55,6 +55,32 @@ jobs:
         run: gradle test
         working-directory: ./java
 
+      # Upload fs-storage JNI libs
+      - name: Install cargo-ndk
+        run: cargo install cargo-ndk
+
+      - name: Add Rust Toolchain
```

Comment:
We can also consider adding cache-support to reduce workflow execution time:

```yaml
      - name: Setup Cargo Cache
        uses: Swatinem/rust-cache@v2
        with:
          save-if: false
```

Reply:
You mean adding it to all the other jobs too?

Reply:
Yes

Reply:
I just added it